test/cppsuite/test_harness/workload/database_operation.h

```diff
@@ -178,7 +178,6 @@ class database_operation {
             if (ret == WT_NOTFOUND) {
                 testutil_check(cursors[i]->reset(cursors[i]));
                 ++i;
-                continue;
```

Comment:
This doesn't do anything since there's no more code under the `if...else` chain.